examples/03_making_adjustments.py

```diff
@@ -2,7 +2,8 @@
 import matplotlib.pyplot as plt
 import numpy as np
 
-import floris.tools.visualization as wakeviz
+import floris.tools.flow_visualization as wakeviz
+import floris.tools.layout_visualization as lf
```

Comment:
Are we ready to commit to "lf" for "layout_visualization"? This decision could be left until the examples are reworked, but it's also relevant to this pull request.

Reply:
I'm switching to layoutviz to match wakeviz

Reply:
while we're at it... do you want to change `wakeviz` to `flowviz`? Or, change flow_visualization.py to wake_visualization.py (I know I was the one to suggest flow_visualization.py, sorry about that). Just seems they should probably be consistent (`wakeviz` with wake_visualization.py or `flowviz` with flow_visualization.py)

Reply:
ha!  I thought of that, decided maybe it's fine but I bet I'll hear about it and indeed!  I'll fix it now